interface/backend/api/tests.py

```diff
@@ -41,19 +41,3 @@ def test_images_metadata_view(self):
             '1.3.6.1.4.1.14519.5.2.1.6279.6001.619372068417051974713149104919/-80.750000.dcm'
         })
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-
-    def test_candidates_mark(self):
-        candidate = CandidateFactory()
-        url = reverse('candidate-mark', kwargs={'candidate_id': candidate.id})
-        response = self.client.get(url)
-        response_dict = response.json()
-        self.assertEqual(response_dict["response"], "Candidate {} was marked".format(candidate.id))
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-
-    def test_candidates_dismiss(self):
-        candidate = CandidateFactory()
-        url = reverse('candidate-dismiss', kwargs={'candidate_id': candidate.id})
-        response = self.client.get(url)
-        response_dict = response.json()
-        self.assertEqual(response_dict["response"], "Candidate {} was dismissed".format(candidate.id))
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
> I have changed it to return the updated candidate.

So how about something like this...

```python
def test_candidates_mark(self):
    candidate = CandidateFactory()
    url = reverse('review-candidate', kwargs={'candidate_id': candidate.id})
    response = self.client.post(url, json.dumps({'review_result': enums.CandidateReviewResult.MARKED}), 'application/json')
    response_dict = response.json()
    self.assertEquals(response_dict, 'SOMETHING')
    self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Reply:
> The thing is - I am not sure what is the test coverage policy in our project at this stage.

I'd say for changes, just try to make sure the previous tests still pass (making modifications to the test cases where necessary).

Reply:
Yes, something like that should do. Also we'll need to update the property on local candidate and use `CandidateSerializer(candidate, context={'request': None})` to make `SOMETHING` :)
So if we need these tests - I can add them.

Reply:
Sounds like a plan :). Doesn't need to be anything complex. Just something that lets you verify from the response that everything worked as expected. Also, since it's `POST` (and no longer `GET`) am I correct in assuming the status code won't be `200` anymore?

Reply:
I think it's still 200 in case of success :)